Misc/NEWS.d/next/Library/2024-01-24-20-51-49.gh-issue-91602.8fOH8l.rst

```diff
@@ -0,0 +1,3 @@
+Added ``filter`` keyword-only parameter to
```

Comment:
Please use stars to mark up param names. Also, I think we should add an entry to What's New. Reusing the wording from the NEWS entry is normally ok.